evap/evaluation/templates/django/forms/widgets/select.html

```diff
@@ -0,0 +1,5 @@
+<select class="form-control" name="{{ widget.name }}"{% include "django/forms/widgets/attrs.html" %}>{% for group_name, group_choices, group_index in widget.optgroups %}{% if group_name %}
+  <optgroup label="{{ group_name }}">{% endif %}{% for option in group_choices %}
+  {% include option.template_name with widget=option %}{% endfor %}{% if group_name %}
+  </optgroup>{% endif %}{% endfor %}
```

Comment:
formatting looks horrible, where did you copypaste that from? :)

Reply:
well, that would be django ;)

Reply:
yes please bring a little sanity into this mess